index.html

```diff
@@ -87,66 +90,110 @@ <h2 class="section__heading__text">
-      <p class="experience__date">December 2023 - March 2024</p>
-      <p class="experience__description">- Write performant and maintainable code. <br />
-      - Communicate and work with multi-disciplinary team of UI designer, UX designer, PM & tester.<br />
-      - Work with team on a real project - coming soon. <br />
-    - Create this portfolio based on project from Figma. <br />
-   </p>
+      <div class="experience__container">
+        <h5 class="experience__content">
+          DARE IT Mentoring Program - Frontend Developer
+        </h5>
+        <p class="experience__date">December 2023 - March 2024</p>
+        <p class="experience__description">
+          - Write performant and maintainable code. <br />
+          - Communicate and work with multi-disciplinary team of UI designer, UX
+          designer, PM & tester.<br />
+          - Work with team on a real project - coming soon. <br />
+          - Create this portfolio based on project from Figma. <br />
+        </p>
       </div>
 
- <div class="experience__container"> 
-      <h5 class="experience__content">University Of Social Sciences - Frontend Developer</h5>
-      <p class="experience__date">March - July 2023</p>
-      <p class="experience__description">- Develop programming skills and
-improve quality of programming code.<br />
-      - Communicate and work with fellow student on Angular project - website to book sits on the plane.<br />
-    - Create a food recipe search app that uses JavaScript and API.  <br />
+      <div class="experience__container">
+        <h5 class="experience__content">
+          University Of Social Sciences - Frontend Developer
+        </h5>
+        <p class="experience__date">March - July 2023</p>
+        <p class="experience__description">
+          - Develop programming skills and improve quality of programming
+          code.<br />
+          - Communicate and work with fellow student on Angular project -
+          website to book sits on the plane.<br />
+          - Create a food recipe search app that uses JavaScript and API. <br />
+        </p>
       </div>
+    </section>
 
-
-
-  
-
-    <!-- <section class="projects">
-      <h2>My projects</h2>
-      1. Dare IT project - coming soon
-      2. Projekt ze studiów 
-      3. Personal website - this project 
-      karty 
-    </section> -->
-<!-- 
- <section class="projects section">
+    <section class="projects section">
       <div class="section__heading">
         <h2 class="section__heading__text">
           <span class="section__container__text--blue">03.</span>
-          Projects</h2>
+          Projects
+        </h2>
       </div>
+      
+      <div class="projects__container">
 
-      <div class="projects__container"> </div>
+        <article class="card">
+          <a href="" target="_blank">
+          <div class="card__header">
+             <img class="card__icon-earth" src="./img/svg/icon-earth.svg"> </>
```

Comment:
jeśli card__header jest tylko po to żeby trzymać w sobie card__icon-earth to możemy zrezygnować i zostawić samo img

Reply:
I wystarczyła by klasa `card__icon` przy `img` wtedy